packages/smooth_ui_library/pubspec.yaml

```diff
@@ -16,6 +16,7 @@ dependencies:
   percent_indicator: ^2.1.5
   flutter_staggered_grid_view: ^0.3.2
   rubber: ^0.4.0
+  shared_preferences: ^0.5.12+4
```

Comment:
I don't think you need SharedPreferences here.
Just use `Theme.of(context)`.

Reply:
I wanted to try something but rolled back, forgot to remove it

Reply:
I checked again. In facts, it is the case that bodyText2 and not bodyText1 is the standard text. Since the color of the BodyText2 has only changed into an accent, no difference was seen in the text without style. I'll add the revised names to the PR right away